extension.js

```diff
@@ -39,9 +39,13 @@ function with_pop_shell(callback) {
     }
 }
 
-var OVERVIEW_WORKSPACES = 0;
-var OVERVIEW_APPLICATIONS = 1;
-var OVERVIEW_LAUNCHER = 2;
+const OVERVIEW_WORKSPACES = 0;
```

Comment:
These were var to prevent a visibility warning when the dock uses them

Reply:
Okay; reverted that part. (At some point I need to learn exactly what the difference between `var` and `let` is; JavaScript is weird.)